e2e/fiat_tf_test.go

```diff
@@ -0,0 +1,529 @@
+package e2e
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"cosmossdk.io/math"
+	fiattokenfactorytypes "github.com/circlefin/noble-fiattokenfactory/x/fiattokenfactory/types"
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+	"github.com/strangelove-ventures/interchaintest/v8"
+	"github.com/stretchr/testify/require"
+)
+
+func TestFiatTFUpdateMasterMinter(t *testing.T) {
```

Comment:
Looks to be missing happy path test here.

Reply:
Added in: https://github.com/noble-assets/noble/pull/368/commits/a278e1e59be0319c50ef61a3a1ff79105c8fa702